### ISSUE TITLE
Satellite streets mapbox  styles failed to render imagery. 

### ISSUE THREAD

**Issue by tnsivanagarajan** [openlayers/ol-mapbox-style#250]
Hello everyone,

I am new to `ol-mapbox-style`. I am trying to add street map and satellite street map as base maps in my project. 

I can successfully load street maps layer(vector tile). But failed to load satellite streets using 
**`ol-mapbox-style`**

below is my usage, I don't know what I am doing wrong. Can anyone help me out on this issue.

```
// const basemap = `https://api.mapbox.com/styles/v1/mapbox/streets-v11?access_token=${MAPBOX_ACCESS_TOKEN}`;
 
const basemap = `https://api.mapbox.com/styles/v1/mapbox/satellite-streets-v11?access_token=${MAPBOX_ACCESS_TOKEN}`;


olms(map, basemap);
```

If I pass streets map as parameter the map is rendered successfully but not rendered if it is satellite streets.

**Comment by ahocevar** [comment]
Thanks for reporting this. Will be fixed with #251.

**Comment by tnsivanagarajan** [comment]
Thank you. Satellite street map is loading.
